src/openforms/registrations/contrib/objects_api/plugin.py

```diff
@@ -92,15 +92,31 @@ def register_submission(
             objecttype_url = objecttype["url"]
 
         with get_objects_client(options["objects_api_group"]) as objects_client:
-            response = execute_unless_result_exists(
-                partial(
-                    objects_client.create_object,
-                    objecttype_url=objecttype_url,
-                    record_data=record_data,
-                ),
-                submission,
-                "intermediate.objects_api_object",
-            )
+            # update or create the object
+            if not (
+                options.get("update_existing_object")
```

Comment:
```suggestion
                options["update_existing_object"]
```

I think it's fine as the serializer as a default value. I'm also wondering if we should add a data migration to add this default value at the DB level? Probably not necessary.